admin/class-taxonomy-presenter.php

```diff
@@ -0,0 +1,317 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class WPSEO_Taxonomy_Presenter
+ */
+class WPSEO_Taxonomy_Presenter {
+
+	/**
+	 * @var array
```

Comment:
Some more phpdoc would be nice. What are the options used for/what do they hold?